tripal_chado/src/Plugin/Field/FieldType/ChadoAdditionalTypeTypeDefault.php

```diff
@@ -581,7 +582,7 @@ public static function discover(TripalEntityType $bundle, string $field_id, arra
     ];
 
     // The parent class adds collection plugin IDs
-    $field_list = self::discoverPostprocess($field_list);
+    $field_list = parent::discoverPostprocess($field_list);
```

Comment:
I think we should keep it with `self` in all of these fields :thinking: If we use parent here and then one day override this method in a field, the child version will be ignored, which could be difficult to debug. What was the reason to change it to parent?

Reply:
Good catch, my mistake, not even sure why I did that 🙃